google-cloud-bigtable/acceptance/bigtable/table_test.rb

```diff
@@ -78,7 +78,7 @@
 
     table = bigtable.create_table(instance_id, table_id) do |cfs|
       cfs.add("cf1", Google::Cloud::Bigtable::GcRule.max_versions(1))
```

Comment:
Could you have cf1 be the one without the rule and cf2 have max versions? Seems more clear to have the default one be first

Reply:
Done